build-tools/linux-systemd-service/mintlayer-wallet-rpc-testnet.service

```diff
@@ -0,0 +1,43 @@
+# For mainnet, change every testnet instance to mainnet
+[Unit]
+Description=Mintlayer Testnet Wallet RPC Daemon
+After=syslog.target network-online.target
+
+# RULES for picking a user:
+#  1. DO NOT use root
+#  2. DO NOT use a user that has root access through sudo, doas, or similar
+[Service]
+User=mintlayeruser
+
+# Type is simple because the wallet-rpc-daemon is blocking
+Type=simple
+
+# Path of the source code OR executable
+WorkingDirectory=/home/mintlayeruser/mintlayer-core
+
+# Use this if you want the wallet-rpc-daemon to be built from source
+# Consider running the `wallet-rpc-daemon` with the `--help` option to see what options are available (authentication, etc)
+# --start-staking-for-account 0 will start staking for account number 0
+ExecStart=cargo run -j 1 --release --bin wallet-rpc-daemon -- testnet /path/to/wallet/file --start-staking-for-account 0
+# If you're using an executable and didn't clone the source, comment the line above, and uncomment below
+#ExecStart=/path/to/wallet-rpc-daemon testnet
+
+Environment="RUST_BACKTRACE=full"
+# Change `info` to `debug` for deeper logs
+Environment="RUST_LOG=info"
+# Only if compiling from source: Add debug info when building
+Environment="RUSTFLAGS=-g"
```

Comment:
I wonder, why would the user need to build it with debug info? To debug something for us?
It's just that it bloats executables significantly, e.g. node-daemon without the debug info is ~24Mb, and with the info it's ~315Mb. Perhaps we should mention that it will increase the size of the executable noticeably.

Reply:
In case there's a problem and we ask for it. I'll add a note.